include/iocore/net/NetVConnection.h

```diff
@@ -322,6 +322,24 @@ class NetVConnection : public VConnection, public PluginUserArgs<TS_USER_ARGS_VC
     return 0;
   }
 
+  /**
+   * Capture TLS handshake byte statistics.
+   *
+   * For TLS connections, captures the bytes read/written during the TLS handshake.
+   * If already captured, returns the stored values.
+   *
+   * @param[out] bytes_in  Bytes received during handshake
+   * @param[out] bytes_out Bytes transmitted during handshake
+   * @return true if values were captured on this call, false if already captured or not applicable
+   */
+  virtual bool
+  capture_tls_handshake_bytes(uint64_t &bytes_in, uint64_t &bytes_out)
```

Comment:
I don't like to see TLS stuff in NetVConnection. `TLSBasicSupport` is probably a better place.

Reply:
Hmmm, I changed the name of this already, it should have been capture_tls_handshake_bytes()